addons/website/views/website_templates.xml

```diff
@@ -1775,7 +1775,7 @@
                  tabindex="-1"
                  role="dialog">
                 <div class="modal-dialog d-flex s_popup_size_full">
-                    <div class="modal-content oe_structure">
+                    <div class="modal-content">
```

Comment:
I wonder if in fact it would not be better to just hide the cookie bar once you enter edit mode and leave the oe_structure. Because right now, if you open the cookie bar and drop snippets... the editor seems broken because you cannot drop anywhere and you cannot really understand that it is because of the cookie bar. And actually if you remove the unique snippet inside the "discrete" and "classic" version, you should be able to add something back (even if it does not work right now but I'd say let's not waste more time on the cookie bar...). That would actually make the fix more stable as well as we should avoid modifying xml in stable version for no critical reason.

What do you think?

Reply:
Fine by me! 😄